src/sentry/monitors/serializers.py

```diff
@@ -178,6 +178,17 @@ def get_attrs(self, item_list, user, **kwargs):
                     trace_ids.append(item.trace_id.hex)
 
             if trace_ids:
+                start_alias, end_alias, trace_id_alias, project_id_alias = (
+                    Columns.TIMESTAMP.value.alias,
+                    Columns.TIMESTAMP.value.alias,
+                    Columns.TRACE_ID.value.alias,
+                    Columns.PROJECT_ID.value.alias,
+                )
+                assert start_alias is not None
+                assert end_alias is not None
+                assert trace_id_alias is not None
+                assert project_id_alias is not None
+
```

Comment:
this is a better fix:

```diff
$ git diff
diff --git a/src/sentry/snuba/events.py b/src/sentry/snuba/events.py
index ee5dc1e16c..a1282eb5fa 100644
--- a/src/sentry/snuba/events.py
+++ b/src/sentry/snuba/events.py
@@ -9,7 +9,7 @@ class Column:
     event_name: Optional[str]
     transaction_name: Optional[str]
     discover_name: Optional[str]
-    alias: Optional[str]
+    alias: str
     issue_platform_name: Optional[str] = None
 
 
```

Reply:
yeah definitely. will have to run it by the SNS team

Reply:
nah just do it -- the typing for the file passes and there's nothing else that uses that class